test/Microsoft.AspNet.Mvc.Localization.Test/project.json

```diff
@@ -0,0 +1,21 @@
+﻿{
+  "compilationOptions": {
+    "warningsAsErrors": "true"
+  },
+  "dependencies": {
+    "Microsoft.AspNet.Mvc": "6.0.0-*",
+    "Microsoft.AspNet.Mvc.Localization" : "6.0.0-*",
+    "Microsoft.AspNet.Mvc.TestCommon": { "version": "6.0.0-*", "type": "build" },
+    "Microsoft.AspNet.Testing": "1.0.0-*",
+    "Microsoft.AspNet.Mvc.Formatters.Xml": "6.0.0-*",
+    "Microsoft.Framework.WebEncoders.Testing": "1.0.0-*",
+    "Moq": "4.2.1312.1622",
+    "xunit.runner.aspnet": "2.0.0-aspnet-*"
+  },
+  "commands": {
+    "test": "xunit.runner.aspnet"
+  },
+  "frameworks": {
+    "dnx451": { }
```

Comment:
Concerning that nothing is tested using `dnxcore50`. That's not a good way to start a new test project, especially because `MvcLocalizationServiceCollectionExtensionsTest` doesn't use `Moq`. Would be even better to test `HtmlLocalizer` as well.